Readme.md

```diff
@@ -30,51 +34,51 @@ If you already have a user account in your Azure Active Directory tenant, you ca
+3. Click the Save option at the bottom of the page.
+4. Once the Save operation has completed, the value of the key appears. Copy the key to the clipboard. **Important: this is the only opportunity you have to access the value of the key, if you don't use it now you'll have to create a new one.**
 
 Leave the browser open to this page. 
 
 ### Step 5:  Configure the sample to use your Azure Active Directory tenant
 
-At this point we are ready to paste into the VS project the settings that will tie it to its entry in your Azure AD tenant. 
+At this point we are ready to paste the configuration settings into the VS project that will tie it to its entry in your Azure AD tenant. 
 
-1. Open the solution in Visual Studio 2013.
-2. Open the `web.config` file.
-3. Find the app key `ida:Password` and replace the value you copied in step 4.
+1. Open the solution in Visual Studio 2013, by double clicking on the WebApp-WebAPI-MultiTenant-OpenIDConnect-DotNet.sln file in the repository you closed in Step 1.
+2. Open the `web.config` file in the TodoListWebApp project, and locate the <appSettings> section.
+3. Find the key `ida:Password` and replace the value in quotes in the `value` attribute with the string you copied in step 4.
 4. Go back to the portal, find the CLIENT ID field and copy its content to the clipboard
-5. Find the app key `ida:ClientId` and replace the value with the CLIENT ID from the Azure portal.
+5. Find the key `ida:ClientID` and replace the value in quotes the `value` attribute with the CLIENT ID from the Azure portal.
 
 ### Step 6:  [optional] Create an Azure Active Directory test tenant 
 
-This sample shows how to take advantage of the consent model in Azure AD to make an application available to any user from any organization with a tenant in Azure AD. To see that part of the sample in action, you need to have access to user accounts from a tenant that is different from the one you used for developing the application. The simplest way of doing that is to create a new directory tenant in your Azure subscription (just navigate to the main Active Directory page in the portal and click Add) and add test users.
+This sample shows how to take advantage of the consent framework in Azure AD to enable an application to be multi-tenant aware, which allows authentication by user accounts from any Azure AD tenant. To see that part of the sample in action, you need to have access to user accounts from a tenant that is different from the one you used for registering the application. A great example of this type of scenario, is an application that needs to allow Office365 user accounts (which are homed in a separate Azure AD) to authenticate and consent access to their Office365 tenant. The simplest way of doing this is to create a new directory tenant in your Azure subscription (just navigate to the main Active Directory page in the portal and click Add) and add test users.
```

Comment:
Not a huge fan of the word "aware" when referring to multi-tenancy.  I would stick to the audience of who can sign into the app.  

Also let's get rid of the Office365 mention.  I like the idea of including an example here, but I really don't want to mention O365 anywhere in our samples.  It will just confuse people.